### PR TITLE
Removing Jss from style

### PR DESCRIPTION
**Pull request by timea-solid**
This is work for https://github.com/SolidOS/solid-ui/issues/533
Please check code. 
This was not tested beyond storybook.

### CODE REVIEW

**Comment by bourgeoa** [comment]
@timea-solid This is what I get running watch on the full stack
It seems the issue are only located in mashlib header

image

File: src/header/index.ts [hunk 1]
```diff
@@ -168,7 +169,13 @@ export function createLoginSignUpButtons () {
  */
 export function createUserMenuButton (label: string, onClick: EventListenerOrEventListenerObject): HTMLElement {
   const button = document.createElement('button')
-  addStyleClassToElement(button, ['header-user-menu__button'])
+  button.setAttribute('style', style.headerUserMenuButton)
+  button.onmouseover = function () {
```
Comment (SharonStrats) [inline src/header/index.ts:173]: praise: nice that it's interactive with the mouse.

**Comment by bourgeoa** [comment]
@timea-solid 
`style.headerUserMenu` is used on L132 and L241 but do not exits in `style`. Is this correct ?

**Comment by timea-solid** [comment]
> @timea-solid `style.headerUserMenu` is used on L132 and L241 but do not exits in `style`. Is this correct ?

@bourgeoa that is ok because the headerUserMenu was empty, had no defined style.

Please retest. I believe now it is pretty good :)

**Review by bourgeoa — APPROVED** [review]
LGTM
I just merged main before testing

**Comment by bourgeoa** [comment]
@timea-solid 
I think you can merge.
The only thing is that you will have to review that there are no issues with the `chat-threads` branch and make the necessary commits if needed.

**Pull request merged**